metrifier/metrifier.go

```diff
@@ -0,0 +1,119 @@
+package metrifier
+
+import (
+	"time"
+
+	"github.com/arquivei/foundationkit/errors"
+	kitprometheus "github.com/go-kit/kit/metrics/prometheus"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	labelErrorCode = "error_code"
+)
+
+// Metrifier is a struct that helps metrify any computation using
+// two Promethus metrics: execution_count and execution_latency_seconds.
```

Comment:
Nit: this documentation can be improved by saying that there's one metrifer per subject being measured. I.e. two distinct output queues will each have its own metrifier.